packages/angular/projects/clr-angular/src/data/datagrid/providers/selection.spec.ts

```diff
@@ -528,7 +549,8 @@ export default function (): void {
 
         it('does not apply trackBy to single selection with no items', () => {
           const emptyItems = new Items(filtersInstance, sortInstance, pageInstance);
-          const selection = new Selection(emptyItems, filtersInstance);
+          const ngZone = TestBed.get(NgZone);
```

Comment:
nit: ngZone should be moved up into the beforeEach

Reply:
Okay, I'll move up the ngZone in the test! But, as @mathisscott said, the test is now checking for mutability. Should I change the test to make it pass? Normally I don't like to change other tests as they have of course been written with a purpose.